ecllibrary/std/DataPatterns/Profile.ecl

```diff
@@ -159,12 +159,12 @@
  *                              quartiles               numeric_lower_quartile
  *                                                      numeric_median
  *                                                      numeric_upper_quartile
- *                              correlations            numeric_correlations
+ *                              %correlations%            numeric_correlations
```

Comment:
Was this changed on purpose (looks like a global change)?

Reply:
There was a correction of an incorrect global find-and-replace that resulted in the %correlations% string appearing.  I missed that one during the correction; will fix.